### PR TITLE
Cross cloud providers tests

### PR DESCRIPTION
**Pull request by tamirdavid1**

### CODE REVIEW

**Review by edeNFed — APPROVED** [review]
Looks great, maybe we should setup some TTL or something on the ECR registry so images are deleted after some time?

File: tests/e2e/helm-chart/chainsaw-test.yaml [hunk 1]
```diff
@@ -51,20 +69,31 @@ spec:
...
     - apply:
         file: 02-install-simple-demo.yaml
+    - script:
+        timeout: 100s
+        content: |
+          # Wait for the pods to be ready
+          kubectl wait --for=condition=ready pod -l app=frontend --timeout=50s
+          kubectl wait --for=condition=ready pod -l app=coupon --timeout=50s
+          kubectl wait --for=condition=ready pod -l app=inventory --timeout=50s
+          kubectl wait --for=condition=ready pod -l app=membership --timeout=50s
```
Comment (blumamir) [inline tests/e2e/helm-chart/chainsaw-test.yaml:93]: isn't it taken care of by `assert-apps-installed.yaml`?

File: tests/e2e/helm-chart/chainsaw-test.yaml [hunk 2]
```diff
@@ -51,20 +69,31 @@ spec:
...
+            kind load docker-image keyval/odigos-demo-frontend:v0.2
+          else
+            echo "Skipping docker pull and kind load for cross-cloud-tests mode"
+          fi
     - apply:
         file: 02-install-simple-demo.yaml
+    - script:
+        timeout: 100s
+        content: |
+          # Wait for the pods to be ready
```
Comment (blumamir) [inline tests/e2e/helm-chart/chainsaw-test.yaml:89]: what about pricing?

**Review by blumamir — APPROVED** [review]
🥇

**Pull request merged**